V1/utils/config.py

```diff
@@ -74,19 +133,139 @@ def find_task_type(task_type_name):
 def set_scheduler(scheduler):
     global scheduling_method
     scheduling_method = scheduler
+    if (str(scheduler) == 'EE' or 'MM' or 'FEE' or 'MSD' or 'MMU' or 'FCFS' or 'MECT' or 'MEET' ):
+        scheduling_method = scheduler
+    else:
+        print('ERROR: Invalid Scheduling Method. FCFS will be set instead.')
+        scheduling_method = 'FCFS'
 
 def get_scheduler():
     global scheduling_method
     return scheduling_method
 
 
+def set_machine_queue_size(size):
+    global machine_queue_size
+    machine_queue_size = 3
```

Comment:
machine_queue_size could be 'inf'. Consider this case as the default value for immediate schedulers.